ToDo/src/main/java/com/todo/demo/Note/Note.java

```diff
@@ -19,11 +20,50 @@
     private String description;
     private Priority priority;
 
-    public Note(NoteThrift note) {
-        this.id = note.getId();
-        this.name = note.getName();
-        this.description = note.getDescription();
-        this.priority = Priority.LOW;
+    public Note() {
+    }
+
+    public Note toNote(NoteThrift noteThrift) {
```

Comment:
name it from fromTNote() and make it static function

Reply:
Cool done